arm64-nobloat-defconfig

```diff
@@ -77,3 +76,5 @@
 # CONFIG_MEDIA_SUPPORT is not set
 # CONFIG_RC_CORE is not set
 CONFIG_IMX_DSP=y
+CONFIG_PINCTRL_MT8195=y
```

Comment:
No sure if it is necessary since I don't set it in my local defconfig.

Reply:
@plbossart  Thanks. if also need to consider sound soc part (machine driver). please also add CONFIG_SND_SOC_MT8195 and  CONFIG_SND_SOC_MT8195_MT6359_RT1019_RT5682. below configs are that I use currently. thanks.  
 CONFIG_SND_SOC_MT8195=y
CONFIG_SND_SOC_MT8195_MT6359_RT1019_RT5682=y
CONFIG_SND_SOC_SOF_OF=y
CONFIG_SND_SOC_SOF_TOPLEVEL=y
CONFIG_SND_SOC_SOF_MTK_TOPLEVEL=y
CONFIG_SND_SOC_SOF_MT8195=y

Reply:
done!

Reply:
thanks.